docs/software/security-protocol-release-notes.md

```diff
@@ -53,6 +53,12 @@ It then follows that:
 
 # List of releases
 
+## v24.0.0 (2025-10-20)
+* `Ledger` - security - Due to a bug in Protocol 23, 478 ledger entries were incorrectly moved from live to hot archive bucket list. 84 of those entries were subsequently restored and got corrupted. See [CAP-76](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0076.md) for more information. 
```

Comment:
Could we clarify that incorrect _entries_ were moved from live to hot? Current wording implies entries were wrongly moved from live to hot, as if they were supposed to stay live. We should also clarify that corruption occurred during archival - current wording implies corruption occurred on restoration.